web-common/src/features/dashboards/filters/measure-filters/MeasureFilterMenu.svelte

```diff
@@ -29,141 +30,132 @@
     },
   } = getStateManagers();
 
+  const initialValues = {
+    dimension: dimensionName,
```

Comment:
It appears in practice this `dimensionName` is an empty string. Thoughts on initializing an actual dimension?

Reply:
It's an empty string only when the measure filter form hasn't yet been submitted. So, when restoring from proto state (or after submission), it does have a value. I think it would be better give it a more explicit Option<string> type, but I'll put that in a different PR.